secantTests/TabsTests/TabsTests.swift

```diff
@@ -145,27 +147,28 @@ class TabsTests: XCTestCase {
         await store.receive(.updateDestination(nil))
 
         await store.receive(.send(.resetForm))
-        
-        await store.receive(.send(.transactionAmountInput(.textField(.set("".redacted))))) { state in
-            state.sendState.transactionAmountInputState.textFieldState.valid = true
-        }
-        
-        await store.receive(.send(.transactionAddressInput(.textField(.set("".redacted))))) { state in
-            state.sendState.transactionAddressInputState.textFieldState.valid = true
-        }
-        
-        await store.receive(.send(.transactionAmountInput(.updateAmount)))
+
+        // TODO: FIXME
```

Comment:
Add a more indicative TODO here, possibly connect it to an issue. Also a commented code is discouraged and should be removed - it always is accessible in git history anyway.

Reply:
Good catch, for now I linked it to the issue.